<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QWidget" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>426</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>426</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 64);
color: rgb(236, 239, 244);
font: 87 12pt &quot;Segoe UI Black&quot;;
</string>
  </property>
  <widget class="QWidget" name="loginPanel" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="adminPwdInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the administrator's password to modify the data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 222, 233);
border: none;
color: rgb(46, 52, 64);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:none;
	background-color: rgb(236, 239, 244);
	color: rgb(46, 52, 64);
}

QPushButton:hover {
	background-color: rgb(46, 52, 64);
	border: 3px solid rgb(236, 239, 244);
	color: rgb(236, 239, 244);
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:none;
	background-color: rgb(236, 239, 244);
	color: rgb(46, 52, 64);
}

QPushButton:hover {
	background-color: rgb(46, 52, 64);
	border: 3px solid rgb(236, 239, 244);
	color: rgb(236, 239, 244);
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelBtnA">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(46, 52, 64);
	border: 3px solid rgb(236, 239, 244);
	color: rgb(236, 239, 244);
}

QPushButton:hover {
	border:none;
	background-color: rgb(236, 239, 244);
	color: rgb(46, 52, 64);
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="uploadPanel" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="loadBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:none;
	background-color: rgb(236, 239, 244);
	color: rgb(46, 52, 64);
}

QPushButton:hover {
	background-color: rgb(46, 52, 64);
	border: 3px solid rgb(236, 239, 244);
	color: rgb(236, 239, 244);
}</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the administrator's password to modify the data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 222, 233);
border: none;
color: rgb(46, 52, 64);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelBtnB">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(46, 52, 64);
	border: 3px solid rgb(236, 239, 244);
	color: rgb(236, 239, 244);
}

QPushButton:hover {
	border:none;
	background-color: rgb(236, 239, 244);
	color: rgb(46, 52, 64);
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loginBtn</sender>
   <signal>clicked()</signal>
   <receiver>AdminPanel</receiver>
   <slot>adminPasswordAuth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>AdminPanel</receiver>
   <slot>adminPasswordClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadBtn</sender>
   <signal>clicked()</signal>
   <receiver>AdminPanel</receiver>
   <slot>loadFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtnA</sender>
   <signal>clicked()</signal>
   <receiver>AdminPanel</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtnB</sender>
   <signal>clicked()</signal>
   <receiver>AdminPanel</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>newFileLoaded(QString)</signal>
  <slot>adminPasswordAuth()</slot>
  <slot>adminPasswordClear()</slot>
  <slot>loadFileName()</slot>
 </slots>
</ui>
